step-templates/octopus-redeploy-previous-release.json

```diff
@@ -0,0 +1,75 @@
+  },
+  "Parameters": [
+    {
+      "Id": "9f8d7b00-417c-406e-8788-326fd67d22b3",
+      "Name": "redeploy.octopus.server.uri",
+      "Label": "Octopus URI",
+      "HelpText": "Provide the URI to your Octopus server.",
+      "DefaultValue": "#{Octopus.Web.ServerUri}",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "7b2ed439-b215-42fe-8971-3839626988fd",
+      "Name": "redeploy.api.key",
+      "Label": "API Key",
+      "HelpText": "Provide an API key with permission to re-deploy the previous version.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Sensitive"
+      }
+    },
+    {
+      "Id": "92461617-cab6-4dc7-8fdd-0fcfc08c5d66",
+      "Name": "redeploy.release.number",
+      "Label": "Release number",
+      "HelpText": "(Optional)\nProvide a release number to re-deploy.  Leave blank to use the immediate previous release number.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "47eaaeaf-e414-4b34-ba7d-7d84715fd2ea",
+      "Name": "redeploy.prompted.variables",
+      "Label": "Prompted variables",
+      "HelpText": "Enter any prompted variables and their values in the format `VariableName::VariableValue`.  Enter one entry per line.\n\nExample: `MyVar::MyValue`",
```

Comment:
Is the `::` double colons standard? I can see this tripping a user up, but it might be better than an `=` or a single `:`

Reply:
The `::` is what was used in an API example we have.  The code was re-used so I didn't need to re-invent the wheel, as it were.  Regarding the suggestion of `:`, would that work with Structured Configuration Variables?

Reply:
A single colon would probably break things. We have seen it break the CLI, the double colon should avoid that.

Reply:
Updated code and instructions to use `=` instead of `::`.

Reply:
@benPearce1 There are some minor code changes, let me know if the PR is still in an approved status.

Reply:
All good. Still approved